adafruit_ble_apple_media.py

```diff
@@ -137,35 +142,58 @@ def __get__(self, obj, cls):
         return 0
 
 class UnsupportedCommand(Exception):
-    pass
+    """Raised when the command isn't available with current media player app."""
 
 class AppleMediaService(Service):
-    """View and control currently playing media. Unimplemented."""
+    """View and control currently playing media.
+
+    Exact functionality varies with different media apps. For example, Spotify will include the
+    album name and artist name in `title` when controlling playback on a remote device.
```

Comment:
I renamed track_title to title, since it seemed like it was referring to the title variable in line 193

Reply:
Thanks!